### ISSUE TITLE
Is it possible to connect sports glasses to a Jetson nano?

### ISSUE THREAD

**Issue by olibartfast** [ActiveLook/Activelook-API-Documentation#7]
Can a Jetson Nano single board computer send information to sports glasses (e.g. Engo 2) using the Activelook APIs?
thanks

**Comment by thomaschapelle** [comment]
Yes, this should be possible. ActiveLook devices (including the Engo 2) communicate over Bluetooth Low Energy (BLE), so any host that can run a BLE central and use the ActiveLook BLE API/SDK can talk to the glasses — a Jetson Nano qualifies as long as it has BLE connectivity.

One thing to note: the Jetson Nano module itself doesn't include built-in Bluetooth
